kits/connect/src/embed-react.tsx

```diff
@@ -41,6 +41,10 @@ export const OpenIntConnectEmbed = React.forwardRef(
             onReady?.()
           }}
           src={url}
+          height={700}
+          width="100%"
+          // Using style for minWidth since iframe props don't accept it.
```

Comment:
WDYM here? 

          {...iframeProps} can include things likes style and className no that is passed to the iframe?

Reply:
Yes, it can include them, but since we didn't want devs to shoot on their feet I thought it would be better to have sizing defined from our side.

Reply:
perhaps change it to height={iframeProps.height? iframeProps.height > 700 iframeProps : 700}

Reply:
Sure that sounds like a better approach will work on it rn